freemocap/gui/qt/widgets/log_view_widget.py

```diff
@@ -133,6 +146,40 @@ def add_log(self, record: logging.LogRecord):
         )
         self.appendHtml(colored_log_entry)
 
+    @QtCore.Slot()
+    def log_progress(self, bar_length: int = 100, cycles_per_update: int = 2500) -> None:
+        if not hasattr(self, "_progress_counter"):
+            self._progress_counter = 0
+
+        progress_marker = "▁▂▃▄▅▆▇▆▅▄▃▂▁"
```

Comment:
aw cute

Reply:
I had a lot of fun trying different ASCII art for this